api.planx.uk/modules/gis/service/index.js

```diff
@@ -61,37 +42,11 @@ export async function locationSearch(req, res, next) {
   }
 }
 
-// Digital Land is a single request with standardized geometry, so remove timeout & simplify query params
+// Planning Data is a single request with standardized geometry, so timeout is not necessary
 export function locationSearchWithoutTimeout(localAuthority, queryParams) {
```

Comment:
nit: Is it simpler to also drop the `withoutTimeout` suffix and comment as it no longer contextually relevant unless contrasted to `locationSearchWithTimeout()`

Reply:
Updated now!